syntax = "proto3";

package protos;

import "google/protobuf/empty.proto";
import "common.proto";


service WorkerService{
   rpc SendRequest(ReplicaRequest) returns(stream ReplicaReply);
    //Ping from the workerNode
   rpc Ping(google.protobuf.Empty) returns (Ack);
}

message ReplicaRequest {
  string prompt = 2;
}

message ReplicaReply {
  string text = 1;
  bool is_error = 2;
}


